crates/solver/src/settlement_submission/submitter/eden_api.rs

```diff
@@ -24,6 +24,7 @@ impl TransactionSubmitting for EdenApi {
         &self,
         tx: TransactionBuilder<DynTransport>,
     ) -> Result<TransactionHandle, SubmitApiError> {
+        tracing::info!("sending transaction to Eden network...");
```

Comment:
Will we be able to match this log statement with the one that is posting the tx hash (to know the Eden vs. flashbots tx hash)? Otherwise maybe just pass a "Tag" into `common::submit_raw_transaction` to make sure it gets printed in the same statement.

Reply:
Fix included in the following PR